qemu/tests/block_hotplug.py

```diff
@@ -64,6 +64,20 @@ def unplug_device(vm, get_disk_cmd, device):
         """
         disks_before_unplug = find_disk(vm, get_disk_cmd)
         device.unplug(vm.monitor)
+        event_status = utils_misc.wait_for(
+            lambda: vm.monitor.get_event(params.get('event_name')),
+            params.get('event_timeout'), 0.0,
+            float(params.get('check_interval')))
+
+        if not event_status:
+            test.fail("Can not get event \"%s\" under %s."
+                      % (params.get('event_name'), params.get('event_timeout')))
+        else:
```

Comment:
you don't need `else` here.
I think it maybe better to assign `params.get('event_name')` to a variable,say `event_name` , so you only need to get it once. 
Same to `params.get('event_timeout')`